samples/wifi/shell/overlay-scan-only.conf

```diff
@@ -0,0 +1,22 @@
+# Scan only using offload API
+CONFIG_WPA_SUPP=n
+# In 9160 default is socket interface
+CONFIG_NET_DEFAULT_IF_ETHERNET=y
+CONFIG_MBEDTLS=n
+CONFIG_NORDIC_SECURITY_BACKEND=n
+# If MbedTLS is enabled, use the standard files
+CONFIG_GENERATE_MBEDTLS_CFG_FILE=y
+
```

Comment:
Related to Kconfigs of the application, noticed that it is asserting heavily in:
ASSERTION FAIL [0] @ WEST_TOPDIR/nrf/lib/dk_buttons_and_leds/dk_buttons_and_leds.c:186

But when I disabled CONFIG_DK_LIBRARY in my application, it was working fine. Is there a conflict with the LEDs/buttons of the nrf9160DK with the nrf7002 EK? If yes, might be good to be documented, at least somehow. Maybe, explicitly put it OFF here with the comment.

Reply:
AFAIK, `nrf7002_ek.overlay` (shield) doesn't have any LED definitions, @bama-nordic can confirm this.

Reply:
Anyhow this is more about nrf9160 and 7002 DTS unrelated to nrf7002 driver, so, closing this, we can take this offline.

Reply:
Correct. There are no LED definitions in the nrf7002_ek.overlay.. wonder if there are any conflicting pin assignments